writer_test.go

```diff
@@ -81,3 +81,39 @@ func TestOpen(t *testing.T) {
 		assert.Equal(t, tt.filenames, names, "Opened unexpected files given paths %v.", tt.paths)
 	}
 }
+
+type testWriter struct {
+	expected []byte
+	t        *testing.T
+}
+
+func (w *testWriter) Write(actual []byte) (int, error) {
+	assert.Equal(w.t, w.expected, actual, "expected writer to write %v, wrote %v", string(w.expected), string(actual))
+	return 0, nil
+}
+
+func (w *testWriter) Sync() error {
+	return nil
+}
+
+func TestOpenWriters(t *testing.T) {
+	tw := &testWriter{[]byte("test"), t}
+
+	w, err := OpenWriters(tw)
+	if err != nil {
+		t.Fatalf("OpenWriters failed, error: %v", err)
+	}
```

Comment:
Instead, please use the `require` package, which is already included as part of testify.

```
require.NoError(t, err, "OpenWriters failed.")
```